src/rfbzero/redox_flow_cell.py

```diff
@@ -122,25 +125,26 @@ def __init__(
         self.time_increment = time_increment
         self.k_mt = k_mt
         self.const_i_ex = F * roughness_factor * self.geometric_area
-        self.n_cls = n_cls
-        self.n_ncls = n_ncls
+        self.num_electrons_cls = num_electrons_cls
+        self.num_electrons_ncls = num_electrons_ncls
 
         self.prev_c_ox_cls = self.c_ox_cls
         self.prev_c_red_cls = self.c_red_cls
         self.prev_c_ox_ncls = self.c_ox_ncls
         self.prev_c_red_ncls = self.c_red_ncls
-        self.delta_ox = 0.0
-        self.delta_red = 0.0
+        self.delta_ox_mols = 0.0
+        self.delta_red_mols = 0.0
 
         self.nernst_const = (R * temperature) / F
 
-        for key, value in {'cls_volume': self.cls_volume, 'ncls_volume': self.ncls_volume, 'k_0_cls': self.k_0_cls,
+        for key, value in {'cls_volume': self.cls_volume, 'ncls_volume': self.ncls_volume,
                            'cls_start_c_ox': self.c_ox_cls, 'cls_start_c_red': self.c_red_cls,
```

Comment:
Thoughts on renaming `cls_start_c_ox` to `c_ox_cls`?

Reply:
done